packages/components/src/components/tooltip/_tooltip.scss

```diff
@@ -514,6 +514,8 @@
     }
 
     &[data-floating-menu-direction='left'] {
+      margin-left: -$spacing-03;
```

Comment:
Looks like this is an issue with negative numbers and css custom properties. We can do either:

`margin-left: calc(#{$spacing-03} * -1)`

or

`margin-left: rem(-8px)`